tests/ui/stable-mir-print/basic_function.stdout

```diff
@@ -0,0 +1,39 @@
+// WARNING: This is highly experimental output it's intended for stable-mir developers only.
+// If you find a bug or want to improve the output open a issue at https://github.com/rust-lang/project-stable-mir.
+fn foo(_0: i32) -> i32 {
+    let mut _0: (i32, bool);
+
+    bb0: {
+        _2 = 1 Add const 1_i32
+    }
+    bb1: {
+        _0 = move _2
+    }
+}
+fn bar(_0: &mut std::vec::Vec) -> std::vec::Vec {
+    let mut _0: std::vec::Vec;
+    let mut _1: &std::vec::Vec;
+    let _2: ();
+    let mut _3: &mut std::vec::Vec;
+
+    bb0: {
+        _3 = refShared1
+    }
+    bb1: {
+        _5 = refMut {
+    kind: TwoPhaseBorrow,
+}2
+    }
+    bb2: {
+        _0 = move _2
+    }
+    bb3: {
+    }
+    bb4: {
+    }
```

Comment:
These blocks don't have successors... Are we not printing successors yet? Can't keep up with what is landed and what is not landed 🤔

Reply:
If that successors PR is not yet landed, then it's ok, we can fix that later.

Reply:
it's not landed yet. https://github.com/rust-lang/rust/pull/118172